demo/client/backend/src/server.js

```diff
@@ -0,0 +1,45 @@
+/*
+* Copyright IBM Corp All Rights Reserved
+*
+* SPDX-License-Identifier: Apache-2.0
+*/
+
+'use strict';
+
+const fs = require('fs');
+const fabricClient = require('./fabricClient');
+const ccRoute = require('./ccRoute');
+const apiRoute = require('./apiRoute');
+const clockauctionRoute = require('./clockauctionRoute');
+
+var express = require('express');
+var app = express();
+app.use(express.json());
+app.use(function (req, res, next) {
+    res.header("Access-Control-Allow-Origin", "*");
+    res.header("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept");
+    res.header("Access-Control-Allow-Methods", "GET, POST");
+    next();
+});
+
+app.get('/',(function(req,res){
+    res.send('Welcome to Spectrum Auction by Elbonia Communication Commission - Enabled by FPC');
+}));
+
+//  different routes defined
+app.use('/api/cc', ccRoute);
+app.use('/api', apiRoute);
+app.use('/api/clock_auction', clockauctionRoute);
```

Comment:
this is different to what marcus has for his mock auction.  I like, though, your path better as it keeps auction agnostic stuff separate for auction-specific stuff which will make later generalization -- the client framework has only few auction-specific stuff and might be very convenient also for other applications .. -- easier ..